src/designSystemInstance/theme/GlobalStyle.tsx

```diff
@@ -224,4 +224,93 @@ export const GlobalStyle = createGlobalStyle`
       }
     }
   }
+
+  /**************************************
```

Comment:
maybe we should create a new global-style instance for sb-specific styles, this wont be needed in the package itself but will be imported when we place all sb-styles in one file. For now its okay but we should find a way to only provide those styles in the storybook instance

Reply:
Created a ticket for that one